### PR TITLE
feat(console): submit form changes action bar

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

feat(console): submit form changes action bar
Requirements:
https://www.figma.com/file/hqAWH3Di8gkiV5TXAt6juO/%F0%9F%8C%B9-%5BAC%5D-Layout-UI-Optimization?node-id=174%3A23222&t=bKjlee8sZDBEwfTV-4

image

## Testing

image

### CODE REVIEW

File: packages/console/src/components/SubmitFormChangesActionBar/index.module.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+@use '@/scss/underscore' as _;
+
+.container {
+  position: sticky;
```
Comment (simeng-li) [inline packages/console/src/components/SubmitFormChangesActionBar/index.module.scss:4]: do we need the sticky layout here? not sure about the design.

File: packages/console/src/components/SubmitFormChangesActionBar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+};
+
+const SubmitFormChangesActionBar = ({ isOpen, isSubmitting, onDiscard }: Props) => {
+  if (!isOpen) {
+    return null;
+  }
+
+  return (
+    <div className={styles.container}>
+      <Button size="medium" title="general.discard" onClick={onDiscard} />
```
Comment (simeng-li) [inline packages/console/src/components/SubmitFormChangesActionBar/index.tsx:17]: nit: can we call discard when isSubmitting?

File: packages/console/src/components/SubmitFormChangesActionBar/index.module.scss (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/components/SubmitFormChangesActionBar/index.module.scss:4]: We do.
Refer to the design:
image

File: packages/console/src/components/SubmitFormChangesActionBar/index.tsx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/components/SubmitFormChangesActionBar/index.tsx:17]: Good call; We should guard this case.

File: packages/console/src/components/SubmitFormChangesActionBar/index.module.scss (hunk 1 above)
Comment (simeng-li) [reply packages/console/src/components/SubmitFormChangesActionBar/index.module.scss:4]: That is "fixed"?

Comment (xiaoyijun) [reply packages/console/src/components/SubmitFormChangesActionBar/index.module.scss:4]: `fixed` will cover the whole page width. We don't desire that.

**Review by simeng-li — APPROVED** [review]
LGTM

**Review by wangsijie — APPROVED** [review]

**Pull request merged**